### PR TITLE
Changes for jurisdiction specific mergetool

### PR DESCRIPTION
**Pull request by hiteshgarg14**

### CODE REVIEW

**Comment by hiteshgarg14** [comment]
Tests will fail until PR-> [Link](https://github.com/opencivicdata/python-opencivicdata/pull/95)

**Comment by hiteshgarg14** [comment]
So the tests are failing because of  error ,     [Link](https://travis-ci.org/openstates/admintools/builds/252715589#L215)
`error: opencivicdata 2.0.0.dev0 is installed but opencivicdata<2.0,>=1.0 is required by {'pupa'}`

I am wondering **how I can set the version to be <2.0?**

**Comment by jamesturk** [comment]
ah I can test this locally & merge this in soon

File: setup.py [hunk 1]
```diff
@@ -12,9 +12,12 @@
       install_requires=[
             'Django>=1.11',
             'pupa>=0.6',
-            'opencivicdata>=1.0',
+            'opencivicdata',
             'flake8',
       ],
+      dependency_links=[
```
Comment (jamesturk) [inline setup.py]: let's drop this one so we don't forget to, we'll make sure there's a new release of that fairly soon

**Comment by hiteshgarg14** [comment]
Finally, Merge Tool is up for testing/use :)

This PR is ready for review.

**Pull request merged**